builders/server.dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Google LLC
```

Comment:
why doesn't he existing service.dockerfile work?

Reply:
Have to copy the assets over

Reply:
(templates)